particles/canonical.manifest

```diff
@@ -14,4 +14,5 @@ import 'Demo/Demo.recipes'
 
 import 'TicTacToe/TicTacToe.recipes'
 // Uncomment to enable native particle recipes
-// import 'Native/native.recipes'
+import 'Native/native.recipes'
```

Comment:
I know this PR does in fact use Kotlin Native... but we do want to move away from referring to 'native particles'. It's ok to merge as it is currently but we should consider what to rename this to, or whether in fact for this case 'native' is ok.

Reply:
I'll rename to 'Languages' folder I think later, and we'll have Java, Kotlin, C++, Rust, AssemblyScript, etc sub-folders.